fe/src/components/Lesson.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+
+const Lesson = () => {
+  const baseURL = 'http://127.0.0.1:8000/api/lesson';
```

Comment:
- a better approach is to create axios base url, so you don't need to call `http://127.0.0.1` everytime.
- store the axios base url in directory **fe/src/apis**

https://axios-http.com/docs/config_defaults